HelpSource/Classes/SequenceableCollection.schelp

```diff
@@ -889,6 +889,137 @@ copymethod:: SimpleNumber -atanh
 copymethod:: SimpleNumber -owensT
 
 
+subsection:: Testing
+
+method:: ==
+
+Overridden by the need to check for equality of each element with its integer index as independent of identity. See link::Classes/Object#Equality and Identity::.
+
+Discussion::
+code::
+~array_a = [1, 2]; ~array_b = [1, 2] // Evaluate this code first!
+::
+
+list::
+## Equality:
+code::
+~array_a == ~array_b // checking for equlity
+::
+teletype::
+-> true
+::
+It first checks if the two objects are identical, then they're not in the same class, then their sizes aren't the same, then there's an unequal element. If they are equal, then their emphasis::hash table indices:: are equal and identical:
+
+code::
+~array_a.hash
+~array_b.hash
+::
+code::
+~array_a.hash ==  ~array_b.hash
+::
+teletype::
+-> true
+::
+code::
+~array_a.hash === ~array_b.hash
+::
+teletype::
+-> true
+::
+
+## Identity
+
+code::
+~array_a === ~array_b // checking for identity
+::
+
+teletype::
+-> false
+::
+They are two different objects allocated at different addresses in virtual memory:
+
+list::
+## code::~array_a::
+
+code::
+~array_a.dump
+::
+
+post window returns:
```

Comment:
usually we just write "returns", without post window.

Reply:
Yes, I know. It sounds too mechanical. However, I have corrected it as you suggested.